calculator.txt

```diff
@@ -6,3 +6,21 @@ define multiply (int a, int b)
 return a*b;
 define divide (int a, int b)
 return a/b;
+define factorial(int n)
```

Comment:
Factorial and evenOdd functions should not be the part of this calculator file. Please add new file say "Programs.txt" and add into it